include/odp/api/spec/classification.h

```diff
@@ -530,11 +530,21 @@ typedef struct odp_cls_stats_capability_t {
  */
 typedef struct odp_cls_capability_t {
 	/** PMR terms supported by the classifier
-	 * A bit mask of one bit for each of odp_pmr_term_t
-	 */
+	 *
+	 * A bit mask of one bit for each of odp_pmr_term_t. */
 	odp_cls_pmr_terms_t supported_terms;
 
-	/** Maximum number of PMR terms */
+	/** Maximum number of single-term PMRs
+	 *
+	 * Depending on the implementation, using several/composite terms for a
+	 * single PMR may end up incurring more than one PMR element from this
+	 * total capacity. */
+	uint32_t max_pmr;
+
+	/** Maximum number of PMRs per CoS */
+	uint32_t max_pmr_per_cos;
+
+	/** Maximum number of terms per PMR */
```

Comment:
Could include term `composite PMR`, so matches better to `odp_cls_pmr_create()` documentation.

Reply:
V3.